app/views/application/_breadcrumb_navigation.html.erb

```diff
@@ -10,7 +10,9 @@
       <div class="govuk-grid-column-two-thirds">
         <div class="govuk-breadcrumbs">
           <ol class="govuk-breadcrumbs__list">
-            <%= render "create_breadcrumb_navigation", nav: nav %>
+            <li class="govuk-breadcrumbs__list-item">
```

Comment:
Huh, I don't think you meant to commit this change, but it does raise a good point: why are the list items a `div`s not `li`s in create_breadcrumb_navigation?

I think this `li` should come out of this file (otherwise we're wrapping *all* of the crumbs in a single `li`), but I wouldn't say no to you changing

`<div class="govuk-breadcrumbs__list-item" aria-current="page">`

to

`<li class="govuk-breadcrumbs__list-item" aria-current="page">`

in _create_breadcrumb_navigation.html.erb in this commit.